docs/src/integrations/observability/blackfire.md

```diff
@@ -1,102 +1,67 @@
 ---
 title: "Blackfire"
 description: |
-  Platform.sh recommends [Blackfire.io](https://blackfire.io/). Blackfire is a full continuous observability solution (monitoring, profiling, and automated performance testing tool). It can be used on Integration, Staging, and Production environments. Blackfire supports PHP and Python.
+  Platform.sh recommends [Blackfire.io](https://blackfire.io/) for monitoring and profiling web sites and applications. From development to test, staging and production Blackfire offers a unique blend of monitoring, often called APM, and profiling features. Blackfire supports PHP and Python.
 ---
 
 {{< description >}}
 
 {{< youtube bS4dzuzkir0 >}}
 
-It grants details information on your PHP code's resources consumption across Wall-Time, CPU, I/O, Memory, Network Calls, HTTP requests and SQL queries.
+Across all of your sites and testing pipelines, get actionable reports and insights, which help you fix issues.
 
-In addition, it can profile your code automatically and notify you whenever your code does not comply with best practices for PHP, Symfony, Drupal, eZPlatform, Typo3 & Magento code performance management.
+Blackfire grants detailed information on your code's resources consumption across Wall-Time, CPU, I/O, Memory, Network Calls, HTTP requests and SQL queries.
 
-For a high level overview and demo of Blackfire, check out the [full video tutorial](https://www.youtube.com/watch?v=-5icUW9pUH8).
-
-## Version
-
-Check the latest versions of the probe and CLI tool on [Blackfire's documentation](https://blackfire.io/docs/up-and-running/upgrade#latest-versions).
-
-## On a Grid plan
-
-### 1. Get your credentials
-
-**Sign up for the free 15 days Premium trial** at [Blackfire.io](https://blackfire.io/pricing) and install the **Blackfire Companion** web browser extension ([Chrome](https://chrome.google.com/webstore/detail/blackfire-companion/miefikpgahefdbcgoiicnmpbeeomffld) or [Firefox](https://addons.mozilla.org/firefox/addon/blackfire/)).
+In addition, it can profile your code automatically and notify you whenever your code does not comply with best practices for PHP, Python and most Open-Source framework's.
```

Comment:
```suggestion
In addition, it can profile your code automatically and notify you whenever your application does not comply with best practices for PHP, Python, and most Open-Source frameworks.
```